extensions/systemd/index.js

```diff
@@ -17,7 +17,15 @@ class SystemdExtension extends cli.Extension {
     }
 
     _setup(argv, ctx, task) {
-        const uid = getUid(ctx.instance.dir);
+        let uid;
+
+        // getUid returns either the uid or null, but can also throw an error
+        try {
+            uid = getUid(ctx.instance.dir);
+        } catch (e) {
```

Comment:
Since we're catching the error and ignoring it, the change to getUID above (throwing a ProcessError instead) won't have any effect.

Not sure that anything really _needs_ to be changed here, but thought it worth mentioning.

Reply:
I tried it without the `try` and `catch` after the changes and it was breaking for me... Can you think of a better way to solve this?